CHANGELOG.develop

```diff
@@ -423,6 +424,16 @@ Benner and Paweł Siudak):
 - Renamed =spacemacs/mplist-get= function to =spacemacs/mplist-get-values=,
   renamed =spacemacs/plist-get= function to =spacemacs/mplist-get-value=, and
   refactored the latter function (thanks to Benjamin Reynolds)
+- Fixed pdumper enabled on ~SPC f e R~ no matter what (thanks to Sylvain Benner)
+- Display time spent in =user-config= in home buffer (thanks to Sylvain Benner)
+- Don't call =user-load= unless we are dumping (thanks to Sylvain Benner)
+- Removed dotspacemacs variable =dotspacemacs-import-env-vars-from-shell= (thanks to Sylvain Benner)
+- Removed dotspacemacs variable =dotspacemacs-improt-env-vars-shell-file-name= (thanks to Sylvain Benner)
+- Rename environment file from =spacemacs.env= to =.spacemacs.env= (thanks to Sylvain Benner)
+- Move location of =.spacemacs.env= file to home or dotdirectory (thanks to Sylvain Benner)
```

Comment:
Environment variable caching is a new feature since the last release (there's an entry under "Hot new feature"), so these fixes probably should be omitted.